frontend/app/components/routing/wp-details/wp-details.controller.test.js

```diff
@@ -28,8 +28,9 @@
 
 /*jshint expr: true*/
 
-describe('WorkPackageDetailsController', function() {
+describe.only('WorkPackageDetailsController', function() {
```

Comment:
F*ck this switch. I'm going to try out https://www.npmjs.com/package/never-say-only to raise an exception when it finds `.only`.